subhub/tests/unit/stripe/utils.py

```diff
@@ -0,0 +1,22 @@
+import os
+import json
+
+from subhub.cfg import CFG
+from subhub import secrets
+from subhub.api.webhooks.stripe.pipeline import StripeWebhookEventPipeline
+
+__location__ = os.path.realpath(os.path.dirname(__file__))
```

Comment:
Using dunder scores in this manner is typically reserved for Python or special class overrides, etc.